CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## [Unreleased 0.3.1]
 
+**Notes for self-hosted etcd**:
+The newly introduced TLS feature for self hosted etcd is a breaking change.
+Updating to this release requires recreation of existing self-hosted etcd cluster.
```

Comment:
Existing self hosted etcd cluster MUST be recreated after updating the etcd operator.